wiki/configuration/3.7/api.html

```diff
@@ -155,6 +155,28 @@ <h2 id="queue">Full Queue output</h2>
             <td><code>limit</code> <small>optional</small></td>
             <td>Number of jobs to display</td>
         </tr>
+        <tr>
+            <td><code>category</code> / <code>cat</code> <small>optional</small></td>
+            <td>Only return jobs for one or more categories (separated by a comma)<br>
+                Use <code>*</code> for Default
+        </td>
+        <tr>
+            <td><code>priority</code> <small>optional</small></td>
+            <td>Only return jobs for one or more priorities (separated by a comma)<br>
+                Some priorities, while valid, are only briefly used for internal logic and set to another priority afterwards.
+                <ul>
+                    <li><code>-100</code> = Default Priority (of category) <span class="label label-info">INTERNAL</span></li>
+                    <li><code>-4</code> = Stop <span class="label label-info">INTERNAL</span></li></li>
```

Comment:
Please remove the `INTERNAL` ones, they are not relevant in this call, even Paused cannot be used in this call.